tests/xfstests/run.pm

```diff
@@ -331,7 +331,9 @@ sub run {
     save_tmp_file('status.log', $status_log_content);
     my $local_file = "/tmp/opt_logs.tar.gz";
     script_run("tar zcvf $local_file --absolute-names /opt/log/");
-    save_tmp_file('opt_logs.tar.gz', Mojo::File->new($local_file)->slurp());
+    script_run("NUM=0; while [ ! -f $local_file ]; do sleep 10; NUM=\$(( \$NUM + 1 )); if [ \$NUM -gt 5 ]; then break; fi; done");
+    my $tar_content = script_output("cat $local_file");
```

Comment:
Why do cat with a tar.gz file? And I watched the video log, there is really got most unreadable messy code. Or else can switch "cat" to "zcat" ?

Reply:
zcat can't pass folder information into next step, I'd like not to show that information but reload those logs into /opt/log/*/, cat is better in my case. but if it could print nothing would be better, because print output waste time, but I couldn't makes it pass content from tar file into parameter without print out.